addons/event/static/src/client_action/event_barcode.xml

```diff
@@ -4,34 +4,23 @@
 
     <t t-name="event.EventScanView">
```

Comment:
Can't you have an ``EventScanViewNoKiosk`` new template (it is a client template so should be good), use the new one in the client action, and let the old one live its life to avoid breaking customizations ?